src/main/java/com/jalasoft/jfc/controller/PdfConverterController.java

```diff
@@ -86,6 +86,7 @@
      * @param height contains integer value.
      * @param pagesToConvert contains number of pdf file pages.
      * @param request contains client data.
+     * @param isMetadata boolean of thumbnail.
```

Comment:
Review this docmentation, you have 2 params: isMetadata and isThumbnail

Reply:
ok